### ISSUE TITLE
Binance gives the wrong price for executed order

### ISSUE THREAD

**Issue by fredfortier** [enigmampc/catalyst#151]
In relation to issue #150 , I noticed that binance gives the wrong price for an executed order. Based on this screenshot, I'd expect Binance to respond to the "Avg" price of the order (0.00000827). Instead, it responds with the limit price of the order. This explains why we don't get a price for market orders.
![screenshot 2018-01-10 22 44 32](https://user-images.githubusercontent.com/1474114/34807631-ddb64968-f657-11e7-9ca3-129e789b9df7.png)

Here is the order object (0.00000827 not in the payload):
```
{  
   'status':'closed',
   'timestamp':1515641853506,
   'price':8.36e-06,
   'datetime':'2018-01-11T03:37:33.506Z',
   'cost':0.0030346799999999997,
   'id':'17210371',
   'remaining':0.0,
   'info':{  
      u'orderId':17210371,
      u'clientOrderId':u'z8xl93tiQH5ZRTOGEtNI9D',
      u'origQty':u'363.00000000',
      u'icebergQty':u'0.00000000',
      u'symbol':u'TRXBTC',
      u'side':u'BUY',
      u'timeInForce':u'GTC',
      u'status':u'FILLED',
      u'stopPrice':u'0.00000000',
      u'time':1515641853506,
      u'isWorking':True,
      u'type':u'LIMIT',
      u'price':u'0.00000836',
      u'executedQty':u'363.00000000'
   },
   'fee':None,
   'side':u'buy',
   'amount':363.0,
   'type':u'limit',
   'symbol':u'TRX/BTC',
   'filled':363.0
}
```
We may fix this on the CCXT side after further investigation.

**Comment by fredfortier** [comment]
So, this was answered in the attached CCXT issue. Binance (and a handful of other exchanges) don't put the trade execution data in the filled order objects. We need to use the `fetch_my_trades()` api to work around this issue. Looking into it.

**Comment by fredfortier** [comment]
Currently testing in live trading before releasing.

**Comment by q8888620002** [comment]
Is binance available on catalyst ? It seems that the ingest function does not work for it.

**Comment by fredfortier** [comment]
Only in live and paper trading modes.

**Comment by fredfortier** [comment]
Fixed on 0.4.6 please re-open if you still notice an issue.
